Vagrantfile

```diff
@@ -43,13 +52,24 @@ Vagrant.configure(2) do |config|
   config.persistent_storage.size = data_disk_size_gb * 1024
 
   config.vm.provision :shell do |shell|
-    shell.path = 'provision/setup-docker.sh'
+    shell.path = 'provision/setup-docker'
```

Comment:
The setup-docker script is a bit more flexible now - see the script for details.  We can modify instructions elsewhere to use that script for setting up physical machines